tgui/packages/tgui/interfaces/StripMenu.tsx

```diff
@@ -306,6 +307,13 @@ export const StripMenu = (props) => {
                       />
                     );
 
+                    if (item.alternate) {
+                      alternateActions = [];
+                      for (const alternateKey of item.alternate || []) {
+                        alternateActions.push(alternateKey);
```

Comment:
Why are we making an array just to copy over the values? This is gonna be the same array